packages/extensionmanager/src/widget.tsx

```diff
@@ -140,7 +140,6 @@ function ListEntry(props: ListEntry.IProperties): React.ReactElement<any> {
                             entry.latest_version
                           )}
                           minimal
-                          small
```

Comment:
On all buttons here could you add the attribute `appearance="stealth"` to reduce visual changes